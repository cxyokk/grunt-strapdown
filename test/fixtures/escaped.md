# escaped test

This file contains a **</xmp>** tag, which is not allowed by strapdown.js and should be escaped into **</×mp>**(letter 'x' to multiplication sign '×').
