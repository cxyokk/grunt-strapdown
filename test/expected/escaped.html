<!DOCTYPE html>
<html>
<title>escaped</title>
<body>
<xmp theme="cyborg" style="display:none;">
# escaped test

This file contains a **</×mp>** tag, which is not allowed by strapdown.js and should be escaped into **</×mp>**(letter 'x' to multiplication sign '×').

</xmp>
<script src="http://strapdownjs.com/v/0.2/strapdown.js"></script>
</body>
</html>
